puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/puppetDeploy.groovy

```diff
@@ -2,9 +2,14 @@ pipeline {
     agent { label 'admin && sshkey' }
 
     options {
-        timestamps()
-        disableConcurrentBuilds()
         ansiColor('xterm')
+        buildDiscarder(logRotator(numToKeepStr: '3'))
+        disableConcurrentBuilds()
+        timestamps()
+    }
+
+    triggers {
+        pollSCM('H/5 * * * *')
```

Comment:
This polls the foreman-infra repo every 10 minutes?

Reply:
every 5 I'd think